doc/source/reference/c-api.types-and-structures.rst

```diff
@@ -6,7 +6,7 @@ Python Types and C-Structures
 
 Several new types are defined in the C-code. Most of these are
 accessible from Python, but a few are not exposed due to their limited
-use. Every new Python type has an associated :c:type:`PyObject *` with an
+use. Every new Python type has an associated :c:type:`PyObject *<PyObject>` with an
```

Comment:
I'd rewrite that as "has a pointer to an associated :c:type:`PyObject` ,,,"? However, I think this whole section is somewhat lacking, what we usually get is a `PyTypeObject` that subclasses the `PyObject` type. I think it would be better to quote something from the Python documentation.